scripts/build.sh

```diff
@@ -82,23 +106,49 @@ git config --global http.https://gopkg.in.followRedirects true
     pushd "$d/soracom" > /dev/null
     echo "Building artifacts ..."
     go generate
-    go get -u github.com/bearmini/go-acl # required to specify some dependencies explicitly as they are imported only in windows builds
-    go get -u golang.org/x/sys/windows
     gofmt -s -w .
-    goxc -bc="$TARGETS" -d=dist/ -pv=$VERSION -build-ldflags="-X github.com/soracom/soracom-cli/soracom/generated/cmd.version=$VERSION"
-
-    # non-zipped versions for homebrew and testing
-    echo "Building artifacts for homebrew (no zip) ..."
-    goxc -bc="$TARGETS" -d=dist/ -pv=$VERSION -build-ldflags="-X github.com/soracom/soracom-cli/soracom/generated/cmd.version=$VERSION" -tasks-=archive-zip,rmbin
-    for distfile in dist/"$VERSION"/*/soracom; do
-       distos=$( echo "$distfile" | awk -F '/' '{print $3}' )
-       mv "$distfile" "dist/$VERSION/soracom_${VERSION}_${distos}"
-    done
-    for distfile in dist/"$VERSION"/*/soracom.exe; do
-       distos=$( echo "$distfile" | awk -F '/' '{print $3}' )
-       mv "$distfile" "dist/$VERSION/soracom_${VERSION}_${distos}.exe"
+
+    mkdir -p "dist/$VERSION/.tmp"
+
+    for GOOS in $TARGETS; do
+      for GOARCH in $ARCHTECTURES; do
+        IDENT="soracom_${VERSION}_${GOOS}_${GOARCH}"
+        BIN_FILENAME=$IDENT
+        if [ "$GOOS" = "$WINDOWS" ]; then
+          BIN_FILENAME="${IDENT}.exe"
+        fi
+
+        go build -ldflags "-X github.com/soracom/soracom-cli/soracom/generated/cmd.version=$VERSION" \
+          -o "dist/${VERSION}/${BIN_FILENAME}"
+        chmod 0755 "dist/${VERSION}/${BIN_FILENAME}"
+
+        mkdir -p "dist/${VERSION}/.tmp/${IDENT}"
+        cp "dist/${VERSION}/${BIN_FILENAME}" "dist/${VERSION}/.tmp/${IDENT}/soracom"
+
+        (
+          cd "dist/${VERSION}/.tmp"
+
+          if [ "$GOOS" = "$LINUX" ]; then
+            tar zcf "${IDENT}.tar.gz" "$IDENT"
+            mv "${IDENT}.tar.gz" ../
+
+            mkdir -p usr/bin/ DEBIAN
+            cp "${IDENT}/soracom" usr/bin/soracom
+
+            DEBIAN_ARCH=$(convert_goarch_to_debian_arch "$GOARCH")
+            build_debian_control "$VERSION" "$DEBIAN_ARCH" > DEBIAN/control
+            dpkg-deb --build . .
```

Comment:
I got this error:
```
./scripts/build.sh: line 140: dpkg-deb: command not found
```
maybe we need to check if dpkg-deb is installed at the beginning of this script with `command -v dpkg-deb > /dev/null 2>&1 ` etc.